include/bsp/iotc_bsp_mem.h

```diff
@@ -32,40 +32,27 @@ extern "C" {
 #endif
 
 /**
- * @function
- * @brief Allocate memory.
+ * @brief Allocates memory and returns a pointer to the allocated block.
  *
  * @param [in] byte_count The number of bytes to allocate.
- *
- * @return A pointer to the beginning of the allocated block.
  */
 void* iotc_bsp_mem_alloc(size_t byte_count);
 
 /**
- * @function
- * @brief Change the size of a memory block.
+ * @brief Changes the size of a memory block and returns a pointer to the
+ * reallocated block.
  *
- * If the new memory block size is smaller than the old memory block
- * size, the Device SDK saves the maximum amount of memory, even if
- * the block moves to a new location. If the new size is larger, the
- * leftover memory is empty.
+ * @details If the new memory block size is smaller than the old memory block,
```

Comment:
Add a comment that in the current form of the SDK this function isn't necessary to be implemented, see the FreeRTOS and Zephyr BSP impls (they are not implemented but still functional). This comment is necessary since this function is quite cumbersome to implement, much harder than the alloc and free functions.

Reply:
Done